### ISSUE TITLE
Quickstart: not authorized to use this access key

### ISSUE THREAD

**Issue by rebitz** [NYUCCL/psiTurk#217]
Hello - I'm having no luck getting psiTurk set up with the AWS credentials. I seem to have working credentials, put them into ~/.psiturkconfig without quotes or anything like that, have logged in to AWS, mTurk requesters and mTurk requesters sandbox with the same credentials, generated a user in AWS with good credentials, tried those, gave AWS my phone number (confirmed via phone) and credit card, and have replaced the version I originally downloaded with the latest (development version) of psiTurk from github (and replaced the config file with known good credentials), and I just keep getting the error:

"The identity contained in the request is not authorized to use this AWSAccessKeyId (string of numbers)"

Is there anything else I can try here?

& here's the rest of it:

WARNING *****************************
Sorry, AWS Credentials invalid.
You will only be able to test experiments locally until you enter
valid credentials in the AWS Access section of ~/.psiturkconfig

***********************************************************
WARNING
Unable to establish connection to AWS.
While your keys may be valid, your AWS account needs a 
subscription to certain services.  If you haven't been asked
to provide a credit card and verified your account using your 
phone, it means your keys are not completely set up yet.
Please refer to 
	http://psiturk.readthedocs.org/en/latest/amt_setup.html
***********************************************************
Sorry, unable to connect to Amazon Mechanical Turk. AWS credentials invalid.

**Comment by deargle** [comment]
>have logged in to AWS, mTurk requesters and mTurk requesters sandbox with the same credentials

Sounds like you might be trying to use the same username/password you use to log in to these services from your browser as your `aws_access_key_id` and `aws_secret_access_key` values? the access keys are strings of characters: the first is 20 chars long and the second is 40

**Comment by rebitz** [comment]
Thanks for the reply, but - I had figured that one out.

Was following the screenshots from http://psiturk.readthedocs.org/en/latest/amt_setup.html carefully.

Any other thoughts?

**Comment by gureckis** [comment]
i believe this is already outdated because amazon switched to a new security model.  @anselmrothe
do you have memory of how to dealt with this?

**Comment by anselmrothe** [comment]
If you generated the AWS credentials recently then the new security model shouldn't be the issue. Also I think they actually still allow old style credentials (The new security model is described by them as a best practice: "AWS best practice by creating and using AWS Identity and Access Management (IAM) users with limited permissions.")

**Comment by rebitz** [comment]
Ok - I got it worked out - I did generate the IAM user (as suggested by AWS) after I had a bit of trouble getting everything set up (on the off chance that amazon was only allowing IAM users for some reason). But - I suspect I had some other problem at that point, which I only made worse by doing this - when I reverted back to the root user account information just now, I was able to log in no problem.

Not sure if I gave my user account insufficient permissions to access the sandbox or what, but definitely have everything working at this point. Thanks for the help & I'm sorry to bug you folks about a silly problem - hopefully this will be helpful if anyone runs into the same problem down the road!
